calico_kubernetes/calico_kubernetes.py

```diff
@@ -97,6 +147,223 @@ def _configure_interface(self):
         print('Finished configuring network interface')
         return ep
 
+    def _configure_profile(self, endpoint):
```

Comment:
You've got this defined twice, in two different places, and you've removed it from its original location.  Could you clean this up so that its back in the original location?

it will make the diff smaller, and thus easier to review.